mods/ra/rules/palettes.yaml

```diff
@@ -127,3 +127,13 @@
 		BaseName: placebuilding
 		BasePalette: player
 		Alpha: 0.65
+	PaletteFromEmbeddedSpritePalette@multiplayer:
+		Name: multiplayer
+		Image: multiplayer
+		Sequence: idle
+		AllowModifiers: false
+	PaletteFromEmbeddedSpritePalette@transitionr:
```

Comment:
That `r` at the very end looks like an accident/copy-paste leftover.

Reply:
Removed the typo.